### ISSUE TITLE
Bug - [Multi typeahead with chips] - Multi type drop down values are not selected using Control+Option+Space

### ISSUE THREAD

**Issue by rnargotr1710** [patternfly/patternfly-react#12015]
**Describe the problem**
The issue is with Menus>Select>Multi type ahead with chips. And this issue is in Patternfly 5

**How do you reproduce the problem?**
1. Reach to the Menus Table>Select>Multi type ahead with chips section
2. Start the voiceover and try to read the drop down values
3. When we try to select the drop down values using Control+Option+Space , values are not being selected. However 

**Expected behavior**
It should select the values using Control+Option+Space, because while Voice Over is reading instructions, it is stating to select values using Control+Option+Space. Drop down remains open but we are not able to select values
List the workaround if there is one.
Values can be selected using Enter. But since Voice Over is reading to select using Control+Option+Space, it should be selected using this option as well

**Screenshots**

<img width="1512" height="982" alt="Image" src="https://github.com/user-attachments/assets/8ddeb512-c723-431f-a5f9-074cd26486aa" />

**What is your environment?**
 iOS

**Comment by thatblindgeye** [comment]
@rnargotr1710 I tested out using VoiceOver in both PatternFly 6 and PatternFly 5, and for both versions I was able to select/deselect an option in the "Multiple typeahead with chips" (in PF6 it was changed to "Multiple typeahead with labels") using the Ctrl + Option + Space method. I also tried changing some VoiceOver settings, though that was more random as nothing immediately jumped out to me as something that could interfere with this; that said I wouldn't be surprised if maybe there's some setting I have set that may explain why you and I are having different experiences here.

I see your screenshot is within a Chrome browser. Can you just confirm the browser you noticed this issue in, as well as browser and OS version? Wondering if anything needs to be updated (I tested in Chrome 141.0.7390.66 on Mac OS 15.7.1)

**Comment by rnargotr1710** [comment]
@thatblindgeye I am working on Chrome Browser 141.0.7390.66 and Mac OS 26.0.1

**Comment by thatblindgeye** [comment]
Hey @rnargotr1710 , I was able to have someone with the same versions as you test things out and they also were able to get it to work using the VO modifier key (Ctrl + Option) + Space. I will leave this issue open for now in case anyone else has ran into a similar problem, but I'm not 100% if there's anything on our end that could be fixed at this moment in time. If you think of or come across anything you feel might be helpful to figuring this out (maybe certain VO settings you're using, or if you spot anything in our code that might cause the issue) definitely feel free to ping us!

**Comment by thatblindgeye** [comment]
@rnargotr1710 For now I'm going to close this issue as we aren't able to reproduce it and haven't seen activity for a few months. If you're sitll having issues and are able to pinpoint what might be causing it on your end please reach out and let us know, though!
